camel/typing.py

```diff
@@ -26,8 +26,10 @@ class RoleType(Enum):
 class ModelType(Enum):
     GPT_3_5_TURBO = "gpt-3.5-turbo"
     GPT_3_5_TURBO_16K = "gpt-3.5-turbo-16k"
+    GPT_3_5_TURBO_1106 = "gpt-3.5-turbo-1106"
```

Comment:
I believe we don't need to introduce a new model. We can just change the value of `GPT_3_5_TURBO` to `"gpt-3.5-turbo-1106"` and then change back to `gpt-3.5-turbo` after Dec 11.

> gpt-3.5-turbo Currently points to gpt-3.5-turbo-0613. Will point to gpt-3.5-turbo-1106 starting Dec 11, 2023.